insolar/noderole.go

```diff
@@ -24,13 +24,17 @@ const (
 	StaticRoleVirtual
 	StaticRoleHeavyMaterial
 	StaticRoleLightMaterial
+	StaticRoleHeavyReplica
```

Comment:
The same situation. We don't have a new role inside a system. Obeserver is a standalone node